### PR TITLE
Improvement: Add issue templates for github issues

### PR DESCRIPTION
**Pull request by jane-rose**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Expected behavior
+<!-- A clear and concise description of what you expected to happen.-->
+
+
+## 🌍  Your Environment
+
+**Platform:**
+<pre><code>
+<!-- ✍️-->
+- Operating System: [Windows 10 x64, MacOS Big Sur, Ubuntu 20.04, etc]
```
Comment (parachvte) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: macOS

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+<!-- ✍️--> A clear and concise description of the problem...
+
+Exception or Error(optional)
```
Comment (parachvte) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```
Exception or Error(optional)
->
Exception or Error (optional)
```

space

Comment (parachvte) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```
Exception or Error(optional)
->
Exception or Error (optional)
```

space

**Comment by jane-rose** [comment]
Updated @parachvte

**Comment by parachvte** [comment]
Nice work 😃

**Pull request merged**